packages/ifpack2/doc/UsersGuide/options.tex

```diff
@@ -302,6 +302,12 @@ \section{Chebyshev}\label{s:Chebyshev}
      eigenvalue, if it is not provided by the user.}
 \cccc{eigen-analysis: iterations}
     {Same as {\tt "chebyshev: eigenvalue max iterations"}, for compatibility with \ml{}.}
+\ccc{chebyshev: eigenvalue tol}
+    {double}
+    {0.0}
```

Comment:
I did not want to enable it by default because I don't think it is robust enough. It works in some of my use cases, but needs a lot more testing before being enabled by default. As such, 0.0 is the preferred value.